src/Components/ProfileListDropMenu.js

```diff
@@ -0,0 +1,141 @@
+import IconButton from "@mui/material/IconButton";
+
+import ListItemButton from "@mui/material/ListItemButton";
+import ListItemIcon from "@mui/material/ListItemIcon";
+import ListItemText from "@mui/material/ListItemText";
+
+import { useEffect, useRef, useState } from "react";
+
+export default function ProfileListDropMenu({ onDelete }) {
+  const [open, setOpen] = useState(false);
+  const anchorRef = useRef(null);
+
+  const handleToggle = () => {
+    setOpen((prevOpen) => !prevOpen);
+  };
+
+  function handleClose(event) {
+    if (anchorRef.current && anchorRef.current.contains(event.target)) {
+      return;
+    }
+    setOpen(false);
+  }
+
+  function handleListKeyDown(event) {
+    if (event.key === "Escape") {
+      setOpen(false);
+    }
+  }
+
+  // return focus to the button when we transitioned from !open -> open
+  const prevOpen = useRef(open);
+  useEffect(() => {
+    if (prevOpen.current === true && open === false) {
+      anchorRef.current.focus();
+    }
+
+    prevOpen.current = open;
+  }, [open]);
+
+  const listItemStyling = {
+    fontSize: "1rem",
+    fontFamily: "interMedium",
+  };
+
+  return (
+    <>
+      {/**********************DROP MENU ICON**********************/}
+      <IconButton
+        ref={anchorRef}
+        id="composition-button"
+        alt="add to watchlist"
+        aria-controls={open ? "composition-menu" : undefined}
+        aria-expanded={open ? "true" : undefined}
+        aria-haspopup="true"
+        onClick={(e) => {
+          handleToggle();
+          e.preventDefault();
+        }}
+        onKeyDown={(e) => {
+          if (e.key === "Enter") handleToggle();
+        }}
+        sx={{
+          color: "inherit",
+          border: "0px solid",
+        }}
+      >
+        <DotsThreeOutlineVertical />
+      </IconButton>
+
+      {/**********************DROP MENU POP-UP**********************/}
+      <Popper
+        open={open}
+        anchorEl={anchorRef.current}
+        role={undefined}
+        placement="bottom"
+        transition
+        style={{ zIndex: "4" }}
+      >
+        {({ TransitionProps, placement }) => (
+          <Grow
+            {...TransitionProps}
+            style={{
+              transformOrigin:
+                placement === "bottom-start" ? "left top" : "left bottom",
+            }}
+          >
+            <Paper elevation={6} onClick={(e) => e.preventDefault()}>
+              <ClickAwayListener onClickAway={handleClose}>
+                <MenuList
+                  autoFocusItem={open}
+                  id="composition-menu"
+                  aria-labelledby="composition-button"
+                  onKeyDown={handleListKeyDown}
+                  sx={{
+                    minWidth: "200px",
+                  }}
+                >
+                  <ListItemButton
+                    divider
+                    onClick={() => {
+                      navigator.clipboard.writeText(window.location.href);
```

Comment:
Nice!  As a follow-up, it might be good to show a Snackbar or something to let the user the copying was successful.

Reply:
Yes!  I was thinking the same thing!  Will incorporate into an upcoming commit.